meta/documents/user_guide/de/settings_guide.md

```diff
@@ -0,0 +1,175 @@
+
+### Lagerpuffer
+Die Einstellung `Lagerpuffer` ist eine nicht negative Ganzzahl, die für jede Artikelvariante für jede Wayfair Lieferanten ID eine reservierte Lagermenge festlegt. Der Pufferbetrag wird abgezogen, nachdem alle anderen Bestandsberechnungen durchgeführt wurden, einschließlich der Aggregation der Bestände für mehrere Plentymarkets Lager, denen dieselbe Wayfair Lieferanten ID zugewiesen wurde.
+
+Um den `Lagerpuffer` zu deaktivieren, lassen Sie diese Einstellung leer oder setzen Sie sie auf `0`.
+
+### Standard-Bestellstatus
+Die Einstellung `Standard-Bestellstatus` bestimmt, in welchem ​​Status sich eine neue Plentymarkets-Bestellung nach der Erstellung durch das Wayfair-Plugin befinden soll.
+
+**Hinweis:** Die Auftragserstellung gilt nicht als Änderung des Auftragsstatus. Wenn Sie den Standardbestellstatus auf einen höheren als die Statuswerte für konfigurierten Plentymarkets-Ereignisse setzen, können einige Ereignisse fehlschlagen.
+
+### Standardversandanbieter **(nicht genutzt)**
+Die Einstellung `Standardversandanbieter` ist eine Legacy-Einstellung, die das Verhalten des Plugins nicht mehr beeinflusst.
+**Wenn diese Einstellung in Ihrem System angezeigt wird, empfiehlt Wayfair dringend, dass Sie Ihr Plugin auf eine neuere Version aktualisieren.**
+
+### Item Mapping Method
+Die Einstellung `Item Mapping Method` bestimmt das Verhalten beim Abgleichen der Artikelvariationen von Plentymarkets mit den Wayfair-Produkten. Es wird verwendet, wenn die Bestandslisten an Wayfair gesendet werden und um die angeforderten Produkte in einer Wayfair-Bestellung auszuwählen. Der Benutzer von Plentymarkets sollte diese Einstellung so konfigurieren, dass sie der Art und Weise entspricht, in der die Elementvariationen ausgefüllt werden.
+
+Standardmäßig wird das Feld `Variationsnummer` für die Zuordnung der Artikel von Wayfair-Bestellungen zu Artikelvariationen von Plentymarkets verwendet.
+
+![variation number field](../../../images/de/variation_number_field.png)
+
+Die anderen Optionen sind `EAN` (Barcode) und `SKU`.
+
+#### Verwenden von EAN (Barcode) als Elementzuordnungsmethode
+Wenn die `Item Mapping Method` auf `EAN` eingestellt ist, sollte jede Artikelvariation in Plentymarkets so eingerichtet werden, dass sie einen Barcode enthält, der die Wayfair-Lieferanten-Teilenummer widerspiegelt, die Wayfair in eingehenden Bestelldaten sendet:
+
+![barcode field](../../../images/de/variation_barcode_field.png)
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >>` Artikel bearbeiten`
+
+2. Suchen Sie nach Artikeln, die auf Wayfair verkauft werden sollen
+
+3. **Klicken Sie für jeden Artikel** auf den Artikel in den Suchergebnissen und dann auf `Varianten` ![plus button](../../../images/common/button_plus.png)
+
+4. **Für jede der Varianten**:

```

Comment:
It may actually be desired but I figured I'd mention that the English document has 4 substeps (i, ii, iii, and iv) and it looks like the English step ii and iii were combined into the German ii step.